File: workspace/pset1/greedy.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>
int main(void)
{
    int coins[] = {25, 10, 5, 1};
    float change;
    int Change;
    int coinsNumber = 0;
    
    do
    {
        printf("How much change is owed?: \n");
        change = GetFloat();
    } while (change < 0);
    
    Change = round(change * 100);
    
    for (int i = 0; i < 4; i++)
    {
        int count = 0;
        count = Change / coins[i];
        Change = Change - ( count * coins[i] );
        coinsNumber = coinsNumber + count;
    }
    
    printf("%i\n", coinsNumber);
}

File: Pset6_Shashank/greedy.py
import cs50

def main():
    while True:
        print("How much change is owed?:")
        change = cs50.get_float()
        if change >= 0:
            break
    
    coinsNumber = 0
    Change = int(round(change * 100))
    
    coinsNumber += Change // 25
    Change %= 25
    
    coinsNumber += Change // 10
    Change %= 10
    
    coinsNumber += Change // 5
    Change %= 5
    
    coinsNumber += Change
    
    print("{}".format(coinsNumber))
    
if __name__ == "__main__":
    main()